webapp/components/team_import_tab.jsx

```diff
@@ -48,7 +48,7 @@ class TeamImportTab extends React.Component {
             <div>
                 <FormattedHTMLMessage
                     id='team_import_tab.importHelp'
-                    defaultMessage="<p>Slack import to Mattermost supports importing of messages in your Slack team's public channels.</p><p>To import a team from Slack, go to <strong>Slack > Team Settings > Import/Export Data > Export > Start Export</strong>.  See <a href='https://docs.mattermost.com/administration/migrating.html#migrating-from-slack'>documentation</a> to learn more.</p><p>To import posts with attached files, see <a href='https://github.com/grundleborg/slack-advanced-exporter'>Slack Advanced Exporter</a> for details.</p>"
+                    defaultMessage="<p>Slack import to Mattermost supports importing of messages in your Slack team's public channels.</p><p>To import a team from Slack, go to <strong>Slack > Team Settings > Import/Export Data > Export > Start Export</strong>.  See <a href='https://docs.mattermost.com/administration/migrating.html#migrating-from-slack' target='_blank'>documentation</a> to learn more.</p><p>To import posts with attached files, see <a href='https://github.com/grundleborg/slack-advanced-exporter' target='_blank'>Slack Advanced Exporter</a> for details.</p>"
```

Comment:
@grundleborg is it possible to change this and **not** use `FormattedHTMLMessage` we are trying to get rid of them and just use `FormattedMessage`

Thanks